### PR TITLE
Include docker logs.

### PR DESCRIPTION
**Pull request by VitalyGushin**
https://github.com/networkservicemesh/deployments-k8s/issues/11229

Add docker container logs to build artifacts

### CODE REVIEW

File: extensions/logs/logs.go [hunk 1]
```diff
@@ -125,6 +126,20 @@ func ClusterDump(suiteName, testName string) {
 			if err != nil {
 				logrus.Errorf("An error while getting cluster dump. Error: %s", err.Error())
 			}
+
+			allContainers, _, _, _ := runner.Run(`docker ps --format '{{.Names}}'`)
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:130]: if we get an error here we could just return

File: extensions/logs/logs.go [hunk 2]
```diff
@@ -125,6 +126,20 @@ func ClusterDump(suiteName, testName string) {
...
+			containers := filterContainers(allContainers)
+
+			for _, container := range containers {
+				_, _, dExitCode, dErr := runner.Run(fmt.Sprintf("docker logs %s > %s/%s.log", container, suitedir, container))
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:134]: artifacts dir is missing. Please have a look as it implemented for cluster dump above

File: extensions/logs/logs.go [hunk 3]
```diff
@@ -54,6 +54,7 @@ type Config struct {
 	WorkerCount          int           `default:"8" desc:"Number of log collector workers" split_words:"true"`
 	MaxKubeConfigs       int           `default:"3" desc:"Number of used kubeconfigs" split_words:"true"`
 	AllowedNamespaces    string        `default:"(ns-.*)|(nsm-system)|(spire)|(observability)" desc:"Regex of allowed namespaces" split_words:"true"`
+	AllowedContainers    []string      `default:"nsc-simple-docker,nse-simple-vl3-docker" desc:"Allowed docker containers" split_words:"true"`
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:57]: Could we use here group of regexp patterns? For example, `(nsc-.*)|(nse-.*)|(nsmgr-.*)|(forwarder-.*)`

**Review by denis-tingaikin — CHANGES_REQUESTED** [review]

File: extensions/logs/logs.go (hunk 2 above)
Comment (VitalyGushin) [reply extensions/logs/logs.go:134]: artifacts dir is included in the suitedir as it done above, I use suitedir

File: extensions/logs/logs.go (hunk 3 above)
Comment (VitalyGushin) [reply extensions/logs/logs.go]: done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**